libs/features/mas/web-components/src/merch-mnemonic-list.js

```diff
@@ -5,8 +5,9 @@ export class MerchMnemonicList extends LitElement {
         :host {
             display: flex;
             flex-direction: row;
-            gap: 10px;
-            margin-bottom: 10px;
+            gap: 5px;
+            margin-bottom: 5px;
+            margin-right: 10px;
```

Comment:
Is this reflected for RTL?

Reply:
Layout is the same for RTL geos in this case.

Reply:
That sounds a bit odd, why is the layout not mirrored? Additionally, could you provide some testing details? I did visit the preview URL, but got a completely different modal than what was added here. I also switched to RTL to see how the modal reacts and I see the label covers the icon:
<img width="966" alt="Screenshot 2024-08-07 at 12 00 55" src="https://github.com/user-attachments/assets/827af4ec-d558-496f-9695-b513cb582ef2">

Reply:
To test the experience you have to click on "See all included apps" on the CC All Apps card:
![Screenshot 2024-08-07 at 11 44 01 AM](https://github.com/user-attachments/assets/e495aede-cf7d-4b6d-9517-5636bd9a012b)

The whole TwP experience is not yet build E2E though, but we decided to merge this WIP since our focus shifted to M@S v2.

Reply:
Ok, I see, thanks for sharing! Not having RTL is still a bit odd, maybe you can open a story to address it in the future.